tools/server/src/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+from django.conf.urls import patterns, include, url
+from django.conf.urls.i18n import i18n_patterns
+from django.contrib import admin
+
+urlpatterns = patterns('',
+    # Examples:
```

Comment:
Thats actually from the django-admin generator which I'd like to keep as standard as possible (so its easier for upgrade)